cif_core.dic

```diff
@@ -19730,8 +19712,7 @@ save_atom_site.label
 _definition.id                          '_atom_site.label'
 loop_
   _alias.definition_id
-         '_atom_site_label'  
-         '_atom_site.id' 
```

Comment:
Is it really OK to remove the _atom_site.id alias? It seems that at some point it was introduced for compatibility purposes with the PDBx/mmCIF dictionary (see [1] ).

[1] https://mmcif.wwpdb.org/dictionaries/mmcif_pdbx_v50.dic/Items/_atom_site.id.html

Reply:
Yes, see my comment below (not sure about the difference between a review comment and a general comment on an issue?)